components/cms-controller-manager/pkg/config/asset_webhook_config_service.go

```diff
@@ -0,0 +1,64 @@
+package config
+
+import (
+	"context"
+	"encoding/json"
+	"github.com/kyma-project/kyma/components/asset-store-controller-manager/pkg/apis/assetstore/v1alpha2"
+	"github.com/pkg/errors"
+	"k8s.io/api/core/v1"
+	apiErrors "k8s.io/apimachinery/pkg/api/errors"
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/apimachinery/pkg/types"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+)
+
+type AssetWebHookConfigMap map[string]AssetWebHookConfig
+
+type AssetWebHookConfig struct {
+	Validations []v1alpha2.AssetWebhookService `json:"validations,omitempty"`
+	Mutations   []v1alpha2.AssetWebhookService `json:"mutations,omitempty"`
+}
+
+type assetWebHookConfigService struct {
+	client Client
+}
+
+//go:generate mockery -name=Client -output=automock -outpkg=automock -case=underscore
+type Client interface {
+	Get(ctx context.Context, key client.ObjectKey, obj runtime.Object) error
+}
+
+//go:generate mockery -name=AssetWebHookConfigService -output=automock -outpkg=automock -case=underscore
+type AssetWebHookConfigService interface {
+	Get(ctx context.Context, namespace, name string) (AssetWebHookConfigMap, error)
+}
+
+func NewAssetWebHookService(client Client) *assetWebHookConfigService {
+	return &assetWebHookConfigService{
+		client: client,
+	}
+}
+
+func (r *assetWebHookConfigService) Get(ctx context.Context, namespace string, name string) (AssetWebHookConfigMap, error) {
```

Comment:
If it is called with hardcoded name and namespace, we don't need to pass them in the method